data-non-standard.js

```diff
@@ -87,14 +87,8 @@ exports.browsers = {
     obsolete: true
   },
   firefox20: {
-    full: 'Firefox 20-27',
-    short: 'FF 20-27',
-    family: 'SpiderMonkey',
-    obsolete: true
-  },
-  firefox28: {
-    full: 'Firefox 28-29',
-    short: 'FF 28-29',
+    full: 'Firefox 20-29',
```

Comment:
It would be nice to keep all of the explicit `firefox28` results, but i don't think it needs to block the PR.